src/EnergyPlus/DXCoils.cc

```diff
@@ -8405,13 +8406,14 @@ Label50: ;
 			// Modify total heating capacity based on defrost heating capacity multiplier
 			// MaxHeatCap passed from parent object VRF Condenser and is used to limit capacity of TU's to that available from condenser
 			if ( present( MaxHeatCap ) ) {
-				TotCap = min( MaxHeatCap, TotCap * HeatingCapacityMultiplier );
+				TotCapAdj = min( MaxHeatCap, TotCap * HeatingCapacityMultiplier );
+				TotCap = min( MaxHeatCap, TotCap );
 			} else {
-				TotCap *= HeatingCapacityMultiplier;
+				TotCapAdj = TotCap * HeatingCapacityMultiplier;
 			}
 
 			// Calculate full load outlet conditions
-			FullLoadOutAirEnth = InletAirEnthalpy + TotCap / AirMassFlow;
+			FullLoadOutAirEnth = InletAirEnthalpy + TotCapAdj / AirMassFlow;
 			FullLoadOutAirHumRat = InletAirHumRat;
```

Comment:
Yes, it does do the same thing as before--use an adjusted capacity for most of the calculations. The adjusted capacity was just stored in `TotCap` before so that any time it was used afterwards the adjusted capacity was used. I replaced all those instances with `TotCapAdj` to explicitly indicate that. The only different calculation is now `DXCoil( DXCoilNum ).ElecHeatingPower`, which uses `TotCap`, which is now the unadjusted capacity. Therefore that line didn't change and it's why you're not seeing it in this diff view.